pkg/apis/minio.min.io/v2/helper.go

```diff
@@ -110,6 +110,27 @@ func GetPodCAFromFile() []byte {
 	return namespace
 }
 
+// GetPodServiceCAFromFile extracts the service-ca.crt certificate in Openshift deployments coming from configmap openshift-service-ca.crt
+func GetPodServiceCAFromFile() []byte {
+	namespace, err := os.ReadFile("/tmp/service-ca/ca.crt")
+	if err != nil {
+		return nil
+	}
+	return namespace
+}
+
+// GetPublicCertFilePath return the path to the certificate file based for the serviceName
+func GetPublicCertFilePath(serviceName string) string {
+	publicCertPath := fmt.Sprintf("/tmp/%s/public.crt", serviceName)
+	return publicCertPath
+}
+
+// GetPrivateKeyFilePath return the path to the key file based for the serviceName
+func GetPrivateKeyFilePath(serviceName string) string {
+	publicKeyPath := fmt.Sprintf("/tmp/%s/private.key", serviceName)
```

Comment:
%s/publicKeyPath/privateKey/

Reply:
I am sorry @shtripat I don't get what you mean

Reply:
@pjuarezd Read from `/tmp/%s/private.key`, named with `publicKeyPath`, so @shtripat request a rename.

Reply:
Yes, rename as `privateKey`

Reply:
ohh I see, okay, renamed